scripts/qa_crowbarsetup.sh

```diff
@@ -3260,6 +3260,8 @@ function addfloatingip
     nova floating-ip-create | tee floating-ip-create.out
     floatingip=$(perl -ne "if(/\d+\.\d+\.\d+\.\d+/){print \$&}" floating-ip-create.out)
     nova add-floating-ip "$instanceid" "$floatingip"
+    # flush arp tables in case this floating ip was recently used for something else
+    arp -d "$floatingip"
```

Comment:
Neutron sends the GARP  when a floating ip is added, see [1]

[1] https://github.com/openstack/neutron/blob/master/neutron/agent/l3/legacy_router.py#L28